roles/setup_ipmi_tools/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Setup IPMI
+  block:
+    - name: Install IPMI
+      ansible.builtin.package:
+        name: ipmitool
+        state: present
+    
+    - name: Install python3-pip
+      ansible.builtin.package:
+        name: python3-pip
+        state: present
+    
+    - name: Install python for the ipmi tool module
+      pip:
+        name:
+          - pyghmi
```

Comment:
You already do this in boot_iso

Reply:
yes i moved that from their this is the draft file